pkg/systemd/shutdown.scss

```diff
@@ -1,32 +1,17 @@
-#shutdown-dialog td {
-  padding-right: 20px;
-  vertical-align: top;
-}
-
-#shutdown-dialog .opt {
-  padding: 1px 10px;
-}
-
-#shutdown-dialog .ct-select,
-#shutdown-dialog .dropdown {
-  min-width: 150px;
-}
-
 #shutdown-group {
   overflow: visible;
 }
 
-#shutdown-dialog textarea {
-    resize: none;
-    margin-bottom: 10px;
+#shutdown-hour,
+#shutdown-minute {
+    width: 3em;
+    margin: 0 0.25em;
 }
 
-#shutdown-dialog input {
-    display: inline;
-    width: 10em;
+textarea {
+    resize: vertical;
```

Comment:
That one you could have done in react only https://www.patternfly.org/v4/components/text-area/#vertically-resizable-text-area

Reply:
oh nice, I totally missed this option. I'll repush

Reply:
No worries from my side.